common/static/common/templates/discussion/customwmd-prompt.underscore

```diff
@@ -0,0 +1,44 @@
+<h4 id="editor-dialog-title"><%= title %></h4>
+<div role="status" id="wmd-editor-dialog-form-errors" class="sr" tabindex="-1"></div>
+<div>
+    <form class="form">
+        <fieldset class="field-group">
+            <legend class="form-group-hd sr"><%= title %></legend>
+            <div class="field <%= uploadFieldClass %>">
```

Comment:
I've added a brief section on Underscore escaping to the wiki page. @robrap will be creating a more in-depth document on ReadTheDocs in the near future, but for now this should help.